etc/docker/enterprise-gateway/start-enterprise-gateway.sh

```diff
@@ -30,12 +30,19 @@ EG_KERNEL_WHITELIST=${EG_KERNEL_WHITELIST:-"'r_docker','python_docker','python_t
 export EG_KERNEL_WHITELIST=`echo ${EG_KERNEL_WHITELIST} | sed 's/[][]//g'` # sed is used to strip off surrounding brackets as they should no longer be included.
 export EG_DEFAULT_KERNEL_NAME=${EG_DEFAULT_KERNEL_NAME:-python_docker}
 
+# Determine whether the kernels-allowed list should be added to the start command.
+# This is conveyed via a 'null' value for the env - which indicates no kernel names
+# were used in the helm chart.
+allowed_kernels_option="--KernelSpecManager.whitelist=[${EG_KERNEL_WHITELIST}]"
+if [ "${EG_KERNEL_WHITELIST}" == 'null' ]; then
+	allowed_kernels_option=""
+fi
```

Comment:
I think it makes more sense to have `allowed_kernels_option` default to "". Then change the if to check if  "${EG_KERNEL_WHITELIST}" != 'null' and set the flag inside the if.

Reply:
Good idea.  I also switched the default value to `null` since I found that the `docker-compose.yaml` script includes the Docker-based kernel specs.